package.json

```diff
@@ -23,11 +23,12 @@
     "gulp.1"
   ],
   "scripts": {
-    "coveralls": "lab -r lcov | coveralls",
     "lint": "eslint . && jscs index.js bin/ lib/ test/",
     "prepublish": "marked-man --name gulp docs/CLI.md > gulp.1",
     "pretest": "npm run lint",
-    "test": "lab test/*.js -cv -I Reflect",
+    "test": "mocha --reporter spec",
```

Comment:
I thought `spec` was the default reporter.  You can use the same set of scripts as https://github.com/gulpjs/vinyl/blob/master/package.json#L22-L26